.github/workflows/tarball.yml

```diff
@@ -0,0 +1,247 @@
+name: CmdStan tarball builder
+
+'on':
+  workflow_dispatch:
+    inputs:
+      tarball_name: 
+        description: 'CmdStan build name'
+        required: true
+        default: ''
+      cmdstan_repo:
+        description: 'CmdStan repo'
+        required: true
+        default: 'stan-dev/cmdstan'
+      cmdstan_branch:
+          description: 'CmdStan branch'
+          required: true
+          default: 'develop'
+      stan_repo:
+        description: 'Stan repo'
+        required: true
+        default: 'stan-dev/stan'
+      stan_branch:
+          description: 'Stan branch'
+          required: true
+          default: 'develop'
+      math_repo:
+        description: 'Math repo'
+        required: true
+        default: 'stan-dev/math'
+      math_branch:
+          description: 'Math branch'
+          required: true
+          default: 'develop'
+      stanc3_repo:
+        description: 'Stanc3 repo'
+        required: true
+        default: 'stan-dev/stanc3'
+      stanc3_branch:
+          description: 'Stanc3 branch'
+          required: true
+          default: 'master'
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    container:
+      image: ocaml/opam2:alpine-3.9-ocaml-4.07
```

Comment:
I don't know ANYTHING about OCaml. So maybe this is an opportunity to learn something.

1) Why use `ocaml/opam2` image (DockerHub has 500k pulls per month) instead of the more popular one `ocaml/opam2-staging` image (DockerHub has 5M pulls per month)?

2) Why alpine instead of Debian? (The image is 855mb vs 770mb)?

3) Stan3 only supports ocaml 4.07 right?

Reply:
1. no particular reason apart from we have used this image with stanc3 for 2+ years and it works so more of a "dont change whats working" approach.

2. Mostly because I typically see Alpine suggested or used in most Ocaml guide I look at.

3. Yes, as of right now. We plan on going to 4.08 but not a huge priority.

Reply:
Ok, thanks!